README.md

```diff
@@ -1,5 +1,11 @@
-# metadata-filter ![Test][WorkflowBadge]
+# metadata-filter ![Test][WorkflowBadge] [![Codacy Badge][CodacyBadge]][Codacy] [![Codacy Coverage Badge][CodacyCoverageBadge]][Codacy]
 
 A module for cleaning up artist, album, and song names.
 
+
+[Codacy]: https://app.codacy.com/project/web-scrobbler/metadata-filter/dashboard
```

Comment:
Does this URL work for you? It says the page not found for me. The URL in the main repo doesn't work too.

https://app.codacy.com/gh/web-scrobbler/metadata-filter/dashboard seems like is working URL.

Reply:
Ahh it could be that they migrated to the new gh structure. It'll be the same for the main repo. Too